### PR TITLE
[fix]add group to see the valuation view, without debugging

### PR DESCRIPTION
**Pull request by olveracuenca**

### CODE REVIEW

**Comment by alan196** [comment]
I think it's better to create a new module to add this feature. It can be called 'stock_valuation_no_developer_mode' or something like that.

File: stock_valuation_layer_by_category/views/stock_valuation_layer.xml [hunk 1]
```diff
@@ -34,5 +33,4 @@
             name="context"
         >{'search_default_group_by_categ_id': 1, 'search_default_group_by_product_id': 1}</field>
     </record>
-
```
Comment (alan196) [inline stock_valuation_layer_by_category/views/stock_valuation_layer.xml]: Don't change other modules

**Review by alan196 — CHANGES_REQUESTED** [review]
Also please add content to DESCRIPTION.rst file

File: stock_valuation_no_developer_mode/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Group to see the valuation view, without debugging
```
Comment (rousseldenis) [inline stock_valuation_no_developer_mode/readme/DESCRIPTION.rst]:
```suggestion
* Change the "Valuation" button visibility in stock pickings to 'Billing' accounting group in order to see it without debug mode.
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM. Please change module description to a more understandable way

**Review by rousseldenis — APPROVED** [review]

**Review by alan196 — APPROVED** [review]
👍 LGTM

**Comment by rousseldenis** [comment]
@olveracuenca Could you just improve commit message like:

[ADD] stock_valuation_no_developer_mode

add group to see the valuation view, without debugging

**Review by alan196 — APPROVED** [review]
👍 

Thanks for the module @olveracuenca

**Pull request merged**